### PR TITLE
[MSCTFIME] Add Turkish (tr-TR) translation

### PR DESCRIPTION
**Pull request by scropool**
## Purpose
Adding the Turkish language file for IME and providing the most accurate translation

### CODE REVIEW

**Review by vsaucing — APPROVED** [review]
should be added

**Comment by binarymaster** [comment]
Cc @reactos/lang-turkish, and also @eersoy93 @Zaryob @mahirgul for review

**Review by JoachimHenze — APPROVED** [review]
Approved. I couldn't spot any accel-collisions.

File: dll/ime/msctfime/lang/tr-TR.rc
Comment (JoachimHenze) [inline dll/ime/msctfime/lang/tr-TR.rc:21]: Note: Most of the time MS Windows tries to avoid &i because it is hardly readable. Are you sure that MS uses &i here (you can prove that with a pic). If not, then chosing a different letter than &i might be preferable.

File: dll/ime/msctfime/lang/tr-TR.rc
Comment (binarymaster) [inline dll/ime/msctfime/lang/tr-TR.rc:13]: None of existing translations use accelerator keys for these items.

Comment (JoachimHenze) [reply dll/ime/msctfime/lang/tr-TR.rc:13]: Good point. Is that a glitch in en-US? Or are the accels unneeded in tr-TR? What does MS do?

Comment (binarymaster) [reply dll/ime/msctfime/lang/tr-TR.rc:13]: I remember checking Windows in en-US and ru-RU locales, they were not underlined there.

Comment (JoachimHenze) [reply dll/ime/msctfime/lang/tr-TR.rc:13]: Thx. Then the turkish guys should strip all of them too!

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
Strip the accels pointed to by Stasm.

File: dll/ime/msctfime/lang/tr-TR.rc
Comment (JoachimHenze) [reply dll/ime/msctfime/lang/tr-TR.rc:21]: Resolved, and superseded by stasms comment.

Comment (vsaucing) [reply dll/ime/msctfime/lang/tr-TR.rc:21]: @scropool Try changing it with "Di&zeyi sonlandır"

**Review by mahirgul — COMMENTED** [review]
It looks good ;)

File: dll/ime/msctfime/lang/tr-TR.rc
Comment (binarymaster) [inline dll/ime/msctfime/lang/tr-TR.rc:18]: I've not asked to remove accelerators for these ones... see en-US

File: dll/ime/msctfime/lang/tr-TR.rc
Comment (scropool) [reply dll/ime/msctfime/lang/tr-TR.rc]: @binarymaster look at new commits.

File: dll/ime/msctfime/lang/tr-TR.rc
Comment (scropool) [reply dll/ime/msctfime/lang/tr-TR.rc]: > I've not asked to remove accelerators for these ones... see en-US

alr i fixed that see new commit 👍

File: dll/ime/msctfime/lang/tr-TR.rc
Comment (JoachimHenze) [reply dll/ime/msctfime/lang/tr-TR.rc]: Resolved.

File: dll/ime/msctfime/lang/tr-TR.rc
Comment (JoachimHenze) [reply dll/ime/msctfime/lang/tr-TR.rc]: Resolved. Thanks.

**Review by JoachimHenze — APPROVED** [review]
Approved now. I will merge it as soon as Stasm will have given his approval as well.

**Review by binarymaster — APPROVED** [review]

File: dll/ime/msctfime/msctfime.rc
Comment (binarymaster) [inline dll/ime/msctfime/msctfime.rc:41]: I somehow overlooked this, it's not sorted right alphabetically (T comes after R)

Comment (JoachimHenze) [reply dll/ime/msctfime/msctfime.rc:41]: Good point! I will fix it. Thx 4 mentioning.

**Pull request merged**
